### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by susithrupasinghe**
* Changes
1. Footer margins added and copyright text alignment fixed
2. Nav bar margins added
3. Homepage body margins added and font size reduced

### CODE REVIEW

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx [hunk 1]
```diff
@@ -3,60 +3,62 @@ import FlatVector from '../../../assets/flat-illustrations/boardpeople.png';
...
-        <div className="col-md-6 col-sm-12">
-          <h2>
-            <strong>Lorem ipsum</strong>
-          </h2>
-          <p className="lead">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi elementum at egestas rutrum commodo ultrices
-            vitae vel. Velit a scelerisque et pharetra, tincidunt pulvinar amet. Tristique morbi enim urna viverra massa
-            vehicula quam.
-          </p>
+  <div className="wrap">
```
Comment (gihanrcg) [inline src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: Why do we wrap a section ?

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css [hunk 2]
```diff
@@ -8,6 +9,10 @@
 }
 
 .lead {
-  font-size: 28px;
-  font-weight: 500;
+  font-size: 22px;
```
Comment (gihanrcg) [inline src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css:12]: Make sure these font sizes match with the rest of the site and the design

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx (hunk 1 above)
Comment (susithrupasinghe) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: Section eka mokkhari reason ekk nisa expand wenawa body ekaata fit wennama. paddings or margins ethulen demmata hari giye ne. Then wrap karala demma harigiye.

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css (hunk 2 above)
Comment (susithrupasinghe) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css:12]: Font size lokui godak.  Anyway whole desgin ekema font size reduce karanna wei maybe. Let me know aiya what you think,

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx (hunk 1 above)
Comment (gihanrcg) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: wrap eka section eka athulen danna barida

Comment (susithrupasinghe) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: ehemath kara but css conflict ekk thiyei kohe hari. eka nisa ethulen reduce karata wedak ne.

Comment (gihanrcg) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: 🤔

Comment (gihanrcg) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx:6]: Normally api sections wrap karanne na. Body eke immediate childs widiyata tynne

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css (hunk 2 above)
Comment (gihanrcg) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.css:12]: I agree with you.

File: src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx (hunk 1 above)
Comment (susithrupasinghe) [reply src/modules/HomeModules/WhatWeDoSection/WhatWeDoSection.jsx]: Wrap eka ethulen demma. awlk ne wage

**Comment by gihanrcg** [comment]
@susithrupasinghe commit looks good. Can you take care of the conflicts ?

**Review by DInuwan97 — APPROVED** [review]

**Pull request merged**